### PR TITLE
[Dygraph] modify code to be suited for launch in 2.0rc

### PR DESCRIPTION
**Pull request by wangxinxin08**

### CODE REVIEW

File: configs/yolov3_reader.yml [hunk 1]
```diff
@@ -52,7 +52,7 @@ TrainReader:
   drop_last: true
   worker_num: 4
   bufsize: 4
-  use_process: false #true
+  use_process: true #true
```
Comment (jerrywgz) [inline configs/yolov3_reader.yml:55]: remove comment

File: tools/train.py [hunk 2]
```diff
@@ -193,7 +197,7 @@ def run():
 
 
 def main():
-    dist.spawn(run)
+    run()
```
Comment (jerrywgz) [inline tools/train.py:200]: move check_gpu, check_version... back here?

File: configs/yolov3_reader.yml (hunk 1 above)
Comment (wangxinxin08) [reply configs/yolov3_reader.yml]: Resolved, Thanks!

File: tools/train.py (hunk 2 above)
Comment (wangxinxin08) [reply tools/train.py]: Resolved, Thanks!

**Review by jerrywgz — APPROVED** [review]
LGTM

**Pull request merged**